src/index.js

```diff
@@ -1 +1,52 @@
 'use strict';
+
+const { Server } = require('http');
+const fs = require('fs');
+const zlib = require('zlib');
+const path = require('path');
+
+const server = new Server();
+
+server.on('request', (req, res) => {
+  if (req.method === 'POST') {
+    const chunks = [];
+
+    req.on('data', (chunk) => {
+      chunks.push(chunk);
+    });
+
+    req.on('end', () => {
+      const body = Buffer.concat(chunks).toString().split('&');
+      const formData = {};
+
+      body.forEach((chunk) => {
+        const [key, value] = chunk.split('=');
+
+        formData[key] = value;
+      });
+
+      res.setHeader('Content-Type', 'application/json');
+      res.end(JSON.stringify(formData));
```

Comment:
Consider creating a function that returns an HTML file with your JSON data, you can use a template string to put your data in the right place in the HTML